tls/oid.c

```diff
@@ -383,14 +382,6 @@ typedef struct {
 
 static const oid_ecp_grp_t oid_ecp_grp[] =
 {
-	{
-		{ ADD_LEN(TTLS_OID_EC_GRP_SECP192R1), "secp192r1",	"secp192r1" },
```

Comment:
Definitions of TTLS_OID_EC_GRP_SECP192R1, TTLS_OID_EC_GRP_SECP224R1, TTLS_OID_EC_GRP_SECP192K1, TTLS_OID_EC_GRP_SECP224K1, and TTLS_OID_EC_GRP_SECP256K1 were left undeleted in the code. As the curves are deleted, those definitions are not needed.